charts/argocd-certificate-refresh/templates/role.yaml

```diff
@@ -2,14 +2,21 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: argocd-cert-role
+  namespace: devtroncd
```

Comment:
Make this configurable from values.yaml

Reply:
In the script, we are deleting secrets and pods of argocd from devtroncd namespace `argocd-secret -n devtroncd`
Does it need to be configured or all the configuration is only valid for Devtron pod
https://github.com/devtron-labs/devtron-utilities/pull/57